src/arch/x86/vm.c

```diff
@@ -123,6 +129,15 @@ int vm_arch_init_platform_device(vm_t *v)
     if (serial_init(&v->serial, &v->io_bus))
         return throw_err("Failed to init UART device");
     virtio_blk_init(&v->virtio_blk_dev);
+    vm_net_init(&v->virtio_net_dev);
```

Comment:
It should be fine to `virtio_net_init()` here directly. Remove `vm_net_init()`.

Reply:
I already remove it